docs/STUDIO_SITE.md

```diff
@@ -0,0 +1,14 @@
+# cellPACK Studio Site
+
+The cellPACK client site allows users to edit example recipes, run packings, and visualize results in the browser. 
+
+* Production site: https://cellpack.allencell.org/
+* GitHub repository: https://github.com/AllenCell/cellpack-client
+
+### Publishing Recipes to Studio Site
+To add new example recipes to the client site, the recipe and its associated metadata must be uploaded to Firebase.
+
+1. Set up access to the staging Firebase account, following [these instructions](https://github.com/mesoscope/cellpack/blob/main/docs/REMOTE_DATABASES.md#firebase-firestore)
+2. Determine which recipe fields should be editable in the client, and create a JSON file specifying these editable fields following the schema outlined in [this example](https://github.com/mesoscope/cellpack/blob/0f140859824086d73edab008ff381b5e5717db8b/examples/client-data/example_editable_fields.json)
+3. Follow cellPACK installation requirements [in README](https://github.com/mesoscope/cellpack?tab=readme-ov-file#installation)
+4. Run the following script to upload necessary files to Firebase: `upload -s -r <path_to_recipe_file> -c <path_to_config_file> -f <path_to_editable_fields_file> -n "Name of Recipe" -o <path_to_simularium_result_file>`
```

Comment:
nit: It might help to update the parameter description to something like <local_path_to_simularium_result_file> to make it clearer that it expects a path such as `out/one_sphere/spheresSST/results_one_sphere_debug_1.0.0_seed_0.simularium`